recipes/gh/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set goname = "github.com/cli/cli" %}
+{% set version = "0.5.4" %}
+
+{% set name = "gh" %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # We must use git here beause the program looks
+  # for the git sha&tag to determine its version
+  - folder: {{ pkg_src }}
+    url: https://{{ goname }}/archive/v{{version}}.tar.gz
+    sha256: eacf8dcc085e0a65a76bbc94b8501fd7c2b37e511dd2e24361c02e80e87ba642
+
+build:
+  number: 0
+  script:
+    - pushd {{ pkg_src }}
+    - make bin/gh
+    - mkdir -p "$PREFIX/bin"                # [unix]
+    - cp bin/gh "$PREFIX/bin/"              # [unix]
+    - mkdir -p "%PREFIX%\Scripts\"          # [win]
+    - copy bin\gh.exe "%PREFIX%\Scripts\"   # [win]
+
+requirements:
+  build:
+    - make     # [unix]
+    - m2-make  # [win]
+    - {{ compiler('go') }}
+
+test:
+  requires:
+    - m2-bash  # [win]
+  commands:
+    - bash -c "{{ name|lower }} --version"
```

Comment:
Why do you need a `bash` call here?

Reply:
Mostly copied from the other golang recipes